### DISCUSSION TITLE
Want implement DeepAr filters during broadcast video

### DISCUSSION THREAD

**Discussion by DevendraGetkart24** [ant-media/Ant-Media-Server#6048] [Announcements]
Hello ,

I want to implement deepAr filters into antmedia android sdk .can you please help me ?

**Comment by burak-58** [comment]
Hi @DevendraGetkart24 ,
I am converting this issue a discussion. After discussing with the members we can get it back we it is necessary.

**Comment by USAMAWIZARD** [comment]
@DevendraGetkart24 Can you please take a look at our sample pages , and code example please let me know if it helps or you are looking for something else

[DeepAR Live Sample](https://antmedia.io/webrtc-samples/deepar-publish-play/)
[DeepAR Code Sample](https://github.com/ant-media/StreamApp/blob/master/src/main/webapp/samples/publish_webrtc_deep_ar_effects_frame.html)
